### PR TITLE
Basic form field

### PR DESCRIPTION
**Pull request by eddielo91**
Got circleci to open up mysql server, but unable to pass puppeteer testing...

### CODE REVIEW

File: test/sum.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const sum = require('./sum');
+
+test('adds 1 + 2 to equal 3', () => {
```
Comment (Windsvept) [inline test/sum.test.js:3]: Nice test haha

Comment (eddielo91) [reply test/sum.test.js:3]: It does the job :)

File: public/index.html [hunk 2]
```diff
@@ -1,11 +1,6 @@
 <html>
   <head>
     <title>Yelp Search Bar</title>
-    <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
```
Comment (Windsvept) [inline public/index.html:4]: +1 for removing those and saving to node modules

File: client/container/ReservationContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Reservation from '../component/Reservation.jsx';
+
+const mapStateToProps = state => ({
+  reservationForm: state.reservationForm
+});
+
+export default connect(mapStateToProps)(Reservation);
```
Comment (Windsvept) [inline client/container/ReservationContainer.jsx]: Redux connect is confusing for me, nice!

**Review by Windsvept — APPROVED** [review]

**Comment by eddielo91** [comment]
Was testing the squash and merge feature since I had so many small commits

File: public/index.html (hunk 2 above)
Comment (eddielo91) [reply public/index.html:4]: Yeah I think importing and exporting looks cleaner

**Pull request merged**